### ISSUE TITLE
Add updated date to search result page to quickly see the age of the latest release

### ISSUE THREAD

**Issue by davsclaus** [sonatype-nexus-community/search-maven-org#69]
Thanks for creating an issue! Please fill out this form so we can be
sure to have all the information we need, and to minimize back and forth.

* **What are you trying to do?**

Would be nice if MavenCentral search also shows the date (yyyy-mm-dd) of the artifacts in the search results, so you can quickly see how old the latest release are.

* **What feature or behavior is this required for?**

* **How could we solve this issue? (Not knowing is okay!)**

* **Anything else?**

I posted on twitter and was asked to create an issue here
https://twitter.com/sonatypeDev/status/1083050808483110912

If you click on each item in the list, there is a detailed page for that artefact where you can see the date (updated). But we would like to see that date quickly in the search result list too. 

See screenshot below

![search](https://user-images.githubusercontent.com/477100/50952816-deb9db80-14b1-11e9-9074-90341ae35842.png)

**Comment by SirMaster** [comment]
@davsclaus thank you again for submitting this. 

We added the updated date, we did use a different format then you suggested, i.e. dd-mmm-yyyy instead of yyyy-mm-dd, this so that we are consistent with the artifacts result page.

We are running some final tests and a build will be on the way soon. When deployed it will look like : 
![screen shot 2019-01-10 at 10 33 14 am](https://user-images.githubusercontent.com/3211896/50997437-66ffa700-14d9-11e9-82bd-645f841859b4.png)

![screen shot 2019-01-10 at 10 32 55 am](https://user-images.githubusercontent.com/3211896/50997438-67983d80-14d9-11e9-8955-230b12338091.png)

For smaller views (like mobile) we will hide it to prevent overcrowding the view.
